.trunignore

```diff
@@ -1,3 +1,3 @@
-examples/xcm/asset_teleportation.eg.ts
+examples/xcm/*.eg.ts
```

Comment:
Do these error currently? Or are they just slow?

Reply:
Let's ignore the XCM examples for now. We can re-enable upon configurability of block times.